scripts/projectview.js

```diff
@@ -0,0 +1,43 @@
+var projectView = {};
+
+projectView.menuHandler = function () {
+  $('li.nav-item').on('click', function(event) {
+    console.log(this);
+    event.preventDefault();
+    $('.article-entry').hide();
+    $('#' + $(this).attr('data-content')).fadeIn();
+  });
+};
+
+projectView.moreLessHandler = function () {
+  $('article#projects').on('click', 'a.expand', function(event) {
+    var $anchor = $(this);
+    event.preventDefault();
+    $anchor.siblings('p.whyGood').toggle();
+    if ($anchor.hasClass('expanded')) {
+      $anchor.html('More &rarr;');
+      $anchor.removeClass('expanded');
+    }
+    else {
+      $anchor.html('Less &larr;');
+      $anchor.addClass('expanded');
+    }
+  });
+};
+
+projectView.filterHandler = function () {
```

Comment:
Recommend renaming this to something like populateFilters, since there does not appear to be any click logic just yet.